doc/Manuals/LoadBalancing/ig-xlb_x-road_external_load_balancer_installation_guide.md

```diff
@@ -0,0 +1,823 @@
+
+The slave nodes use the `keyconf.xml` in read-only mode: no changes made from the admin UI are persisted to disk. Slaves
+reload the configuration from disk periodically and apply the changes to their running in-memory configuration.
+
+
+###### 2.3.1.3 Other server configuration parameters from `/etc/xroad/*`
+| Data                                 | Replication          | Replication method                                 |
+| ------------------------------------- | -------------------- | -------------------------------------------------- |
+| other server configuration parameters | **replicated**       |  `rsync+ssh`  (scheduled)                          |
+
+The following configurations are excluded from replication:
+* `db.properties` (node-specific)
+* `postgresql/*` (node-specific keys and certs)
+* `globalconf/` (syncing globalconf could conflict with `confclient`)
+* `conf.d/node.ini` (specifies node type: master or slave)
+
+
+#### 2.3.2 Non-replicated state
+
+###### 2.3.2.1 `messagelog` database: not replicated
+
+The messagelog database is not replicated. Each node has its own separate messagelog database. **However**, in order to
+support PostgreSQL streaming replication (hot standby mode) for the serverconf data, the serverconf and messagelog
+databases must be separated. This requires modifications to the installation (a separate PostgreSQL instance is needed
+for the messagelog database) and has some implications on the security server resource requirements as a separate
+instance uses some memory.
+
+
+###### 2.3.2.2 OCSP responses from `/var/cache/xroad/`: not replicated
+
+The OCSP responses are currently not replicated. Replicating them could make the cluster more fault tolerant but the
+replication cannot simultaneously create a single point of failure. A distributed cache could be used for the responses.
+
+
+## 3. X-Road Installation and configuration
+
+This chapter details the complete installation on a high level, with links to other chapters that go into the details.
+
+You can set up the cluster manually, or use the provided Ansible playbook \[[SS-CLUSTER](#12-references)\] if it suits
+your purposes.
+
+### 3.1 Prerequisites
+
+In order to properly set up the data replication, the slave nodes must be able to connect to:
+* the master server using SSH (tcp port 22), and
+* the master `serverconf` database (e.g. tcp port 5433).
+
+
+### 3.2 Master installation
+
+1. Install the X-Road security server packages using the normal installation procedure or use an existing standalone node.
+2. Stop the xroad services.
+3. Create a separate PostgreSQL instance for the `serverconf` database (see section
+   [4. Database replication setup](#4-database-replication-setup) for details).
+4. Change `/etc/xroad/db.properties` to point to the separate database instance:
+   * `serverconf.hibernate.connection.url` : Change the url port number from `5432` to `5433` (or the port you specified)
```

Comment:
The point in 3.3.4 is to change it to what the master db password already is. On the master side the password should already be set to the correct one, no?